package.json

```diff
@@ -130,6 +130,7 @@
     "jsonld": "~1.1.0",
     "jsonld-signatures": "https://github.com/Chocobozzz/jsonld-signatures#rsa2017",
     "lodash": "^4.17.10",
+    "loglevel": "^1.6.3",
```

Comment:
Please don't add another dependency. If you want to manage log levels, create your own helper (or use winston)

Reply:
OK, I'll look for another option. I do believe this functionality - i.e. user-selectable verbosity - should be included in all the tools as it is makes the CLI-tools far more usable in cron jobs etc. Would you accept a patch to the other tools which includes this? I made this for my own use but I'm sure there are others who want to control their PT-instance(s) using cron jobs and other automated tools which generally prefer less chatty/more concise output from client programs.

Reply:
> Would you accept a patch to the other tools which includes this

Sure, but I think you should just use winston with a Console transporter